examples/lighting-app/silabs/efr32/README.md

```diff
@@ -278,9 +278,8 @@ combination with JLinkRTTClient as follows:
 
     Here is an example with the CHIPTool:
```

Comment:
```suggestion
    Here is an example with chip-tool:
```
I know it's not the part you are changing, but there are totally different tools in the tree that are called CHIPTool, and this one is _not_ called that.